configure.ac

```diff
@@ -375,6 +376,27 @@ AC_COMPILE_IFELSE([AC_LANG_PROGRAM([[
 )
 CXXFLAGS="$TEMP_CXXFLAGS"
 
+case $host in
+  *mingw*)
+    ;;
+  *)
+    TEMP_CXXFLAGS="$CXXFLAGS"
+    CXXFLAGS="$CXXFLAGS $AVX512_CXXFLAGS"
+    AC_MSG_CHECKING(for AVX512 intrinsics)
+    AC_COMPILE_IFELSE([AC_LANG_PROGRAM([[
```

Comment:
Use `AC_LINK_IFELSE` here instead

Reply:
Add a little code to make mingw generate xmm register more than 16 is ok? @luke-jr